src/servers/main.cc

```diff
@@ -240,8 +234,6 @@ std::vector<Option> options_{
      "The port for the server to listen on for HTTP requests."},
     {OPTION_HTTP_HEALTH_PORT, "http-health-port",
      "The port for the server to listen on for HTTP Health requests."},
-    {OPTION_HTTP_THREAD_COUNT, "http-thread-count",
```

Comment:
You can't remove this flag as that would break backwards compatibility, but you can put a note where it is parsed below and just say that it is currently unused.

Reply:
Sure, I'll do that